webapp/src/components/post_menu_actions/attach_comment_to_issue/attach_comment_to_issue.jsx

```diff
@@ -56,16 +56,6 @@ export default class AttachCommentToIssuePostMenuAction extends PureComponent {
                     {this.getLocalizedTitle()}
                 </button>
             );
-        } else {
```

Comment:
1/5 consolidate lines 43-48, smething like
```
    const conn = this.props.connected || {};
    if (this.props.isSystemMessage || !conn.connected) {
            return null;
    }

    let content;
    ...
```

Reply:
I like this idea as it removes the extra if statement.  I think we will eventually refactor these post menu options components and end up doing the same for the create issue post menu modal.  Submitting change soon.